python/ray/rllib/agents/ars/ars.py

```diff
@@ -294,6 +301,10 @@ def _train(self):
         # Set the new weights in the local copy of the policy.
         self.policy.set_weights(theta)
 
+        # Now sync the filters
+        for new_filter in filters:
+            self.policy.get_filter().sync(new_filter)
```

Comment:
For example, I think the sync() here will overwrite the local state, to merge the filters you actually want apply_changes() like in FilterManager.